apps/omg_watcher_info/test/omg_watcher_info/db/eth_event_test.exs

```diff
@@ -14,7 +14,7 @@
 
 defmodule OMG.WatcherInfo.DB.EthEventTest do
   use ExUnitFixtures
-  use ExUnit.Case, async: true
+  use ExUnit.Case, async: false
```

Comment:
why did it become sync test?

Reply:
so we've learned that all ecto tests that use `SQL.Sandbox.checkout/1` and shared mode are supposed to be synchronous. and because we previously used `--trace` that was enforced. now that I've deleted the trace flag, tests started failing because of that. 

```
:ok = SQL.Sandbox.checkout(WatcherInfo.DB.Repo)
SQL.Sandbox.mode(WatcherInfo.DB.Repo, {:shared, self()})
```
https://hexdocs.pm/ecto_sql/Ecto.Adapters.SQL.Sandbox.html#module-shared-mode